clang/test/Analysis/builtin_bitcast.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_analyze_cc1 -triple x86_64-unknown-unknown -verify %s \
-// RUN:   -analyzer-checker=core,debug.ExprInspection
+// RUN:   -analyzer-checker=debug.ExprInspection -analyzer-disable-checker=core
```

Comment:
Why do you disable core?
That is not supposed to be disabled. Checkers or the engine may crash if core is disabled.

Reply:
Oh, I was going to just disable `core.FixedAddressDereference`